plugins/mcreator-localization/help/default/entity/condition_solid_bounding_box.md

```diff
@@ -1,2 +1 @@
-While the returned value is true, the entity's bounding box is solid.
-This feature is used by boats, for example.

```

Comment:
You can keep this line so that there is an example of this feature being used in the game.